### PR TITLE
test(ssr): add test for bookends with text concat

### PR DESCRIPTION
**Pull request by nolanlawson**
## Details

Adds some tests for tricky edge cases with slots, `lwc:if`, and adjacent text.

## Does this pull request introduce a breaking change?

- 😮‍💨 No, it does not introduce a breaking change.

## Does this pull request introduce an observable change?

- 🤞 No, it does not introduce an observable change.

### CODE REVIEW

File: packages/@lwc/engine-server/src/__tests__/fixtures/slot-not-at-top-level/lwcIf-with-adjacent-text-nodes/shadow/modules/x/outer/outer.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { LightningElement } from 'lwc';
+
+export default class extends LightningElement {
+    static renderMode = 'shadow';
+
+    isTrue = true
+    isFalse = false
```
Comment (nolanlawson) [inline packages/@lwc/engine-server/src/__tests__/fixtures/slot-not-at-top-level/lwcIf-with-adjacent-text-nodes/shadow/modules/x/outer/outer.js:7]:
```suggestion
```

**Review by wjhsf — APPROVED** [review]
